hefquin-engine/src/test/java/se/liu/ida/hefquin/engine/federation/catalog/FederationDescriptionReaderTest.java

```diff
@@ -162,4 +165,97 @@ public void tpfFMWithoutRequiredProperty() {
 		assertEquals(expectedErrorMessage, actualErrorMessage);
 	}
 
+	@Test
+	public void vocabularyMappingFileIsLoadedFromClasspathResource() {
+		final String turtle =
+				"PREFIX fd:     <http://www.example.org/se/liu/ida/hefquin/fd#>\n"
+						+ "PREFIX ex:     <http://example.org/>\n"
+						+ "\n"
+						+ "ex:dbpediaSPARQL\n"
+						+ "      a            fd:FederationMember ;\n"
+						+ "      fd:interface [ a                  fd:SPARQLEndpointInterface ;\n"
+						+ "                     fd:endpointAddress <http://dbpedia.org/sparql> ];\n"
+						+ "                     fd:vocabularyMappingsFile \"dbpedia/vocabularyMappings.nt\".";
+
+		final Model fd = ModelFactory.createDefaultModel();
+		final RDFParserBuilder b = RDFParser.fromString( turtle );
+		b.lang( Lang.TURTLE );
+		b.parse( fd );
+
+		final FederationCatalog cat = FederationDescriptionReader.readFromModel( fd );
+		final FederationMember fm = cat.getFederationMemberByURI( "http://dbpedia.org/sparql" );
+		assertNotNull( fm.getVocabularyMapping() );
+	}
+
+	@Ignore("Disabled since it uses live web test")
+	@Test
+	public void vocabularyMappingFileIsLoadedFromRemoteURL() {
```

Comment:
I don't think that we need this test. From the perspective of the `FederationDescriptionReader` code that is tested here, it tests the same code path as the test with the local file. The only difference is in how the internally-used Jena code (`RDFDataMgr.loadGraph(path)`) behaves, but perhaps we can assume that it behaves as advertised.